test/src/test/java/org/corfudb/infrastructure/log/StreamLogFilesTest.java

```diff
@@ -543,7 +453,7 @@ public void testPrefixTrim() {
 
         // Write 50 segments and trim the first 25
         final long numSegments = 50;
-        final long filesPerSegment = 3;
+        final long filesPerSegment = 1;
```

Comment:
why did you change this?

Reply:
Currently on master, we have 3 files per segment. For example, for segment 1, we have 1.log          1.log.pending  and 1.log.trimmed. This test used to verify that the 3 files were being generated. Now that we removed those 2 files, the assertion would fail.